storage/db-tool/src/replay_on_archive.rs

```diff
@@ -183,6 +185,8 @@ impl Verifier {
         let (start, limit) =
             Self::get_start_and_limit(&arc_db, config.start_version, config.end_version)?;
         set_layout_caches(true);
+        // Paranoid checks are done async if enabled.
+        set_async_runtime_checks(config.paranoid_type_checks);
```

Comment:
nit: probably better to move after setting paranoid?

Reply:
done